test/integration/statistics_announcement_test.rb

```diff
@@ -0,0 +1,145 @@
+require "test_helper"
+require "gds_api/test_helpers/publishing_api_v2"
+
+class StatisticsAnnouncementTest < ActiveSupport::TestCase
+  self.use_transactional_fixtures = false
+
+  setup do
+    DatabaseCleaner.clean_with :truncation
+    stub_any_publishing_api_call
+  end
+
+  test "it gets published to the Publishing API when saved" do
+    statistics_announcement = build(:statistics_announcement)
+    presenter = PublishingApiPresenters.presenter_for(statistics_announcement)
+
+    statistics_announcement.save!
+
+    expected = presenter.content.merge(
+      public_updated_at: Time.zone.now.as_json
+    )
+
+    assert_publishing_api_put_content(statistics_announcement.content_id,
```

Comment:
@gpeng Don't let it block this PR. We can auto-correct these after, if the outcome of alphagov/styleguides#84 dictates a new cop. :+1: